src/compiler/core/src/commonTest/kotlin/community/flock/wirespec/compiler/core/CompileFullEndpointTest.kt

```diff
@@ -166,21 +166,41 @@ class CompileFullEndpointTest {
             |  }
             |}
             |
+            |package community.flock.wirespec.generated
+            |
+            |import community.flock.wirespec.kotlin.Wirespec
```

Comment:
It would be nice if the imports are not needed to not have them here

Reply:
We could do full qualified class names. Only I think for the Wirespec import it becomes a bit verbose